{
    "GSD": {
        "alias": "CVE-2023-38003",
        "id": "GSD-2023-38003"
    },
    "namespaces": {
        "cve.org": {
            "CVE_data_meta": {
                "ASSIGNER": "psirt@us.ibm.com",
                "ID": "CVE-2023-38003",
                "STATE": "PUBLIC"
            },
            "affects": {
                "vendor": {
                    "vendor_data": [
                        {
                            "product": {
                                "product_data": [
                                    {
                                        "product_name": "Db2 for Linux, UNIX and Windows",
                                        "version": {
                                            "version_data": [
                                                {
                                                    "version_affected": "=",
                                                    "version_value": "10.5, 11.1 ,11.5"
                                                }
                                            ]
                                        }
                                    }
                                ]
                            },
                            "vendor_name": "IBM"
                        }
                    ]
                }
            },
            "data_format": "MITRE",
            "data_type": "CVE",
            "data_version": "4.0",
            "description": {
                "description_data": [
                    {
                        "lang": "eng",
                        "value": "IBM Db2 for Linux, UNIX and Windows (includes Db2 Connect Server) 10.5, 11.1, and 11.5 could allow a user with DATAACCESS privileges to execute routines that they should not have access to.  IBM X-Force ID:  260214."
                    }
                ]
            },
            "generator": {
                "engine": "Vulnogram 0.1.0-dev"
            },
            "impact": {
                "cvss": [
                    {
                        "attackComplexity": "LOW",
                        "attackVector": "NETWORK",
                        "availabilityImpact": "HIGH",
                        "baseScore": 7.2,
                        "baseSeverity": "HIGH",
                        "confidentialityImpact": "HIGH",
                        "integrityImpact": "HIGH",
                        "privilegesRequired": "HIGH",
                        "scope": "UNCHANGED",
                        "userInteraction": "NONE",
                        "vectorString": "CVSS:3.1/AV:N/AC:L/PR:H/UI:N/S:U/C:H/I:H/A:H",
                        "version": "3.1"
                    }
                ]
            },
            "problemtype": {
                "problemtype_data": [
                    {
                        "description": [
                            {
                                "lang": "eng",
                                "value": "264 Permissions, Privileges, Access Controls"
                            }
                        ]
                    }
                ]
            },
            "references": {
                "reference_data": [
                    {
                        "name": "https://www.ibm.com/support/pages/node/7078681",
                        "refsource": "MISC",
                        "url": "https://www.ibm.com/support/pages/node/7078681"
                    },
                    {
                        "name": "https://exchange.xforce.ibmcloud.com/vulnerabilities/260214",
                        "refsource": "MISC",
                        "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/260214"
                    }
                ]
            },
            "source": {
                "discovery": "UNKNOWN"
            }
        },
        "nvd.nist.gov": {
            "configurations": {
                "CVE_data_version": "4.0",
                "nodes": []
            },
            "cve": {
                "CVE_data_meta": {
                    "ASSIGNER": "psirt@us.ibm.com",
                    "ID": "CVE-2023-38003"
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "en",
                            "value": "IBM Db2 for Linux, UNIX and Windows (includes Db2 Connect Server) 10.5, 11.1, and 11.5 could allow a user with DATAACCESS privileges to execute routines that they should not have access to.  IBM X-Force ID:  260214."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": []
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://www.ibm.com/support/pages/node/7078681",
                            "refsource": "",
                            "tags": [],
                            "url": "https://www.ibm.com/support/pages/node/7078681"
                        },
                        {
                            "name": "https://exchange.xforce.ibmcloud.com/vulnerabilities/260214",
                            "refsource": "",
                            "tags": [],
                            "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/260214"
                        }
                    ]
                }
            },
            "impact": {},
            "lastModifiedDate": "2023-12-04T02:58Z",
            "publishedDate": "2023-12-04T01:15Z"
        }
    }
}